package auth_service.exception;

public class UserNotFoundException {
}
